workers/scanner.go

```diff
@@ -54,6 +61,10 @@ var (
 			Dev:      1 * database.MiB,
 			Testing:  1 * database.MiB,
 		}).(int)
+	// MaxRepairingSkylinks defines the maximum number of skylink which scanner
+	// is allowed to wait on (wait for their health to reach 0). When we reach
+	// this number we'll stop pinning new skylinks until we go under it again.
+	MaxRepairingSkylinks = int32(3)
```

Comment:
I might of missed it, but is this used? If not, fine to remove as a follow up.

Reply:
Yes, I've missed this during cleanup. I was working on a more flexible approach to this but it turned out it's not viable, so I scrapped it. It seems there are some leftovers.